### ISSUE TITLE
Opening and closing the Window repeatedly will continuously increase memory usage.

### ISSUE THREAD

**Issue by Soarts** [nwjs/nw.js#5504]
Hi.

Opening and closing the Window repeatedly will continuously increase memory usage.
It is occured When it is set chromium-args : --mixed-context .
Even though implementing  global.gc() code, there is no change in the accumulated memory.
If --mixed-context option is removed and tested, the memory is normally terminated.

This is test code.

package.json
"main": "index.html",
"js-flags": "--expose-gc",
"chromium-args": "--mixed-context",

HTML :
<button onclick="window.open('./blank.html', '_blank')">window open</button>
<button onclick="window.open('./index.html', '_self')">reload</button>

Test Environment
ㆍchromium : "54.0.2840.71"
ㆍnw : "0.18.2"
ㆍnw-commit-id : "a820695-68f77db-ae7e389-2021126"
ㆍOS : Windows7, Windows10 32bit

Is it a bug? or am i using it wrong?

**Comment by rogerwang** [comment]
Thanks for reporting. Could you please provide the full package.json and index.html?

**Comment by Soarts** [comment]
sure. thank you.
[src_test_memory.zip](https://github.com/nwjs/nw.js/files/612675/src_test_memory.zip)

**Comment by renemarty** [comment]
Any news on this? It's a serious bug of nw... I'm analysing to migrate a huge app to electron only due to this memory issue. Any advice or comment will be appreciated.

**Comment by Christywl** [comment]
I try the attached test, here is my result on Windows 10 64bit with nwjs-sdk-v0.22.0.
[beginning]
![image](https://cloud.githubusercontent.com/assets/20039821/25325979/b35c35b2-2900-11e7-8683-9be89acfa0ca.png)
[open the windows with `window.open` and then close it for 30 times]
![image](https://cloud.githubusercontent.com/assets/20039821/25326038/e42439ba-2900-11e7-9e34-39f5a30b3d1c.png)
[open the windows with `window.open` and then close it for 60 times]
![image](https://cloud.githubusercontent.com/assets/20039821/25326046/ef517b18-2900-11e7-8791-7ac2fd24fb0c.png)

**Comment by rogerwang** [comment]
@Christywl are you trying with reporter's sample?

**Comment by rogerwang** [comment]
@renemarty could you please provide a sample? Memory leak in the application doesn't necessarily mean there is a leak bug in NW.

**Comment by Christywl** [comment]
@rogerwang , yes, as I said in the previous comment , I used the attached test _**src_test_memory.zip**_

**Comment by bq-hentai** [comment]
@Christywl  @rogerwang How can we solve this problem ?

**Comment by Christywl** [comment]
The result on Windows 10 with nwjs-sdk-v0.24.4:
[beginning]
![image](https://user-images.githubusercontent.com/20039821/29810966-6ef092be-8cd4-11e7-8268-0f48368ae349.png)

[click 'window open' button and then close it for 40 times]
![image](https://user-images.githubusercontent.com/20039821/29810951-6194fb64-8cd4-11e7-8a1a-b8b872fa5ea9.png)


[click 'window open' button and then close it for 70 times]
![image](https://user-images.githubusercontent.com/20039821/29810934-54788ad6-8cd4-11e7-8b10-0354ccd8698e.png)
Need wait for rogerwang to investigate.

**Comment by bq-hentai** [comment]
@Christywl fyi, I test Electron and no memory leak...

**Comment by daemon-7** [comment]
nw.js 0.30.1
node v10.0.0
chromium 66.0.3359.117
os win7 and win10

open child window， close， open chid window， close ... repeat， memory leak， the same as them
@rogerwang
